Prac_05/state_names.py

```diff
@@ -0,0 +1,27 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+Mischa Burgess
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+
+state_code = input("Enter short state: ")
+while state_code != "":  # while state code is not empty
+    if state_code.islower():  # checks if entered in lowercase
```

Comment:
consider changing the case when input, this will allow odd entries such as Tas

Reply:
updated it, thanks! :)